apps/dg/resources/dg.css

```diff
@@ -1525,6 +1541,14 @@ html.dg-embedded-mode .dg-data-tip-text {
     color: black;
 }
 
+.slick-multiline-cells .slick-cell {
```

Comment:
Curious why -webkit-line-clamp required dynamic setting, since it is easier set in code. Also, I cannot find `display: -webkit-box` documented anywhere. I see that it is comes from the article, but is it on a standards track and is it likely to persist with the same meaning in browsers?

Reply:
For `-webkit-line-clamp`, its value is a numeric value indicating the number of lines to clamp to, which varies with row height and so can't be set statically in CSS. As described in the article, it's not apparently on any standards track at this time, but it has been adopted by all the major browsers. Without it, there basically isn't a clean solution to the problem of putting the ellipsis where it belongs.

Reply:
One other note: I used that particular set of CSS because it's what was described in the article, including the comment that "Firefox supports this now, exactly this way (with the -webkit- prefixes and all)." It's conceivable that the line-clamping would work with `display: flex`, for instance (presumably, that's the way it will work eventually), but I didn't try it.

Reply:
Yes, I saw that in the article. Thanks.